src/main/java/org/threadly/litesockets/NoThreadSocketExecuter.java

```diff
@@ -221,8 +225,8 @@ protected void shutdownService() {
       if(selector != null && selector.isOpen()) {
         selector.close();
       }
-    } catch (IOException e) {
-
+    } catch (Exception e) {
+      ExceptionUtils.getExceptionHandler().handleException(e);
```

Comment:
getExceptionHandler() may return null if none is set.

ExceptionUtils.handleException() is another option (but if no ExceptionHandlerInterface is set, and an UncaughExceptionHandler is set, it will be called instead...it does prioritize an ExceptionHandlerInterface first though).

http://threadly.github.io/threadly/javadocs/3.8.0/org/threadly/util/ExceptionUtils.html#handleException-java.lang.Throwable-

Sorry to keep throwing it back at you, but I do think it looks good besides this.